test/coverband/configuration_test.rb

```diff
@@ -86,4 +86,23 @@ def setup
       end
     end
   end
+
+  test 'use_oneshot_lines_coverage' do
+    Coverband::Collectors::Coverage.instance.reset_instance
+    refute Coverband.configuration.use_oneshot_lines_coverage
+
+    Coverband.configuration.stubs(:one_shot_coverage_implemented_in_ruby_version?).returns(true)
```

Comment:
Can we avoid stubbing here? I think it would be possible by doing each of these assertions within a separate example/test_method.